apps/web/src/components/features/search/SearchBar.tsx

```
@@ -0,0 +1,163 @@
+              />
+            </div>
+
+            {showSuggestions && (
+              <ul className="absolute w-full mt-1 bg-background border rounded-md shadow-md z-20 max-h-48 overflow-y-auto">
+                {suggestions.map((suggestion) => (
+                  <li
+                    key={suggestion}
+                    onClick={() => selectSuggestion(suggestion)}
+                    className="p-2 hover:bg-muted cursor-pointer"
+                    role="option"
+                    tabIndex={0}
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<option>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)